srcpkgs/tessen/template

```diff
@@ -0,0 +1,17 @@
+# Template file for 'tessen'
+pkgname=tessen
+version=2.2.1
+revision=1
+hostmakedepends="scdoc"
+short_desc="Interactive menu to autotype and copy pass and gopass data"
+maintainer="reedts <j.reedts@gmail.com>"
+license="GPL-2.0-only"
+homepage="https://git.sr.ht/~ayushnix/tessen"
+distfiles="https://git.sr.ht/~ayushnix/tessen/refs/download/v${version}/tessen-${version}.tar.gz"
+checksum=31d510d5ffac5825bc7e63f13283c5d432eb102db2d2483ee937c9135c8cfc04
+
+do_install() {
+	make PREFIX=/usr DESTDIR=${DESTDIR} install
```

Comment:
would this work?
```suggestion
build_style=gnu-makefile

post_install() {
```

Reply:
Yes that would work too. However, when using `build_style=gnu-makefile` you'd get a build error because it will try to build `make all`, which in this case does not exist. If you want to use `gnu-makefile` I'd suggest to unset the build function like this:

```sh
do_build() {
    :
}
```

Reply:
it does the default target in `do_build`, and I don't see any issues with letting [that target](https://git.sr.ht/~ayushnix/tessen/tree/v2.2.1/item/Makefile#L22-25) run, it shouldn't fail

Reply:
@classabbyamp my bad, I meant "build warning" rather than error. Nevertheless I updated the template.